js/calendar.js

```diff
@@ -192,7 +192,7 @@ function loadEvent(event) {
 		document.getElementById('calendar-event').style.display = "block"; //show
 	} else {
 		//null event
-		console.log("ERROR: event is null");
+		console.error("ERROR: event is null");
```

Comment:
Sorry, one final thing, we don't need the comment about keeping this, and also, mainly stylistic, we don't need to print "ERROR" as part of the error message.

Reply:
Fixed